nixos/modules/programs/nist-feed.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.nist-feed;
+in
+{
+  options = {
+    programs.nist-feed = {
+      enable = mkEnableOption (mdDoc "NIST Feed, which notifies you about the newest published CVEs");
+      package = mkPackageOption pkgs "nist-feed" { };
+      extraArgs = mkOption {
+        type = types.str;
+        default = "-l -s CRITICAL";
+        description = mdDoc ''
+          Arguments to provide to NIST-Feed, see a full list at https://github.com/D3vil0p3r/NIST-Feed/blob/main/README.md#nist-feed
+        '';
+      };
+
+      onCalendar = mkOption {
+        type = types.str;
+        default = "*:0/30";
+        description = mdDoc ''
+          How often NIST-Feed executes.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+    systemd.user.services.nist-feed = {
+        wantedBy = [ "default.target" ];
+        description = "A notification daemon for CVEs";
+        path = [ pkgs.curl pkgs.busybox ];
```

Comment:
Maybe we can remove these, now that the program has all its dependencies baked in?

```suggestion
```